fn-fdk.js

```diff
@@ -128,9 +128,9 @@ function sendResult (ctx, resp, result) {
     isJSON = true
   }
 
-  let headers = ctx.responseHeaders
-  for (let key in headers) {
-    if (headers.hasOwnProperty(key)) {
+  const headers = ctx.responseHeaders
+  for (const key in headers) {
+    if (Object.prototype.hasOwnProperty.call(headers, key)) {
```

Comment:
These seems familiar. This is more secure/robust  than the original because the header object version of the 'hasOwnProperty' method could have be meddled with by a framework lib.

Was that the intention of the change? Did you change this due to a known issue, or, just as part of general improvements?